spec/steps/solicitor_declaration/form_spec.rb

```diff
@@ -10,13 +10,16 @@
     }
   end
 
-  let(:application) { instance_double(Claim, update!: true, 'status=': true) }
+  let(:application) { create(:claim, :complete) }
```

Comment:
Not in the PR alone but I see the use of the term application and claim being used interchangeably. This concerns me as with crm4 coming along we may need to differentiate 😬

Reply:
Yeah I think we need to have a wider chat about this as even I'm getting confused now on what the difference is between and claim and an application - as they seem to be interchangeable/definition is changing a lot of the time